main/http_server/axe-os/src/app/components/home/home.component.ts

```diff
@@ -107,27 +121,34 @@ export class HomeComponent {
     };
 
 
-    this.info$ = interval(5000).pipe(
+    this.info$ = interval(2000).pipe(
```

Comment:
Is this adding a lot of value increasing the frequency of api calls?

Reply:
Hmm, yeah maybe its not worth the extra load/work.  I thought about just making the UI feel more responsive, but unless you bring it down even more I don't think it feels enough different to change it